tardis/plasma/properties/nlte_rate_equation_solver.py

```diff
@@ -937,7 +935,7 @@ def ion_matrix(ion_coefficients, atomic_number, ion_number):
     offdiag = np.zeros(atomic_number)
     index = ion_coefficients.index
     for i in index:
-        offdiag[i] = ion_coefficients.loc[i]
+        offdiag[i] = float(ion_coefficients.loc[i].iloc[0])
```

Comment:
Does it need to have nested `.loc.iloc`?

Reply:
Yep, to make sure it only extracts a single value and not a series. I will replace iloc with values for consistency and it is slightly faster than iloc.